zdm/cosmology.py

```diff
@@ -29,7 +29,7 @@
 DEF_Omega_b=0.044
 DEF_Omega_b_h2=0.0224 #Planck says 0.0224, WMAP 0.02264
 # hubble constant in current epoch
-DEF_H0 = 67.4 #km s^-1 Mpc^-1 #planck
+DEF_H0 = 67.74 #km s^-1 Mpc^-1 #planck15 used in frb.igm
```

Comment:
Can we instead read this directly from the FRB library? We should to remove hard-coded values. So e.g. load frb definitions and define DEF_H0 as the value from there

Reply:
Okay, will do!